sale_order_digitized_signature/readme/INSTALL.rst

```diff
@@ -0,0 +1,5 @@
+This module depends on :
```

Comment:
This should be removed, since `web_widget_digitized_signature` is no more required.